### PR TITLE
Add code layout descriptions

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,11 +58,27 @@ make openapi
...
+| [db/migrations](./db/migrations/) | Database Migrations                                                                                                                                                                            |                                                                                                                                                                            |
+| [pkg/api](./pkg/api)                 | Api Structures that are used for handling data within our Api Handlers                                                                                                                         |
+| [pkg/config](./pkg/config)                      | Config loading and application bootstrapping code                                                                                                                                              |
+| [pkg/dao](./pkg/dao)                         | Database Access Object.  Abstraction layer that provides an interface and implements it for our default database provider (postgresql.  It is separated out for abstraction and easier testing |
```
Comment (rverdile) [inline README.md:69]: "(postgresql " is missing the closing parentheses

File: README.md [hunk 2]
```diff
@@ -58,11 +58,27 @@ make openapi
...
+|-----------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [api](./api/)                     | Openapi docs and doc generation code                                                                                                                                                           |
+| [db/migrations](./db/migrations/) | Database Migrations                                                                                                                                                                            |                                                                                                                                                                            |
+| [pkg/api](./pkg/api)                 | Api Structures that are used for handling data within our Api Handlers                                                                                                                         |
```
Comment (rverdile) [inline README.md:67]: Api should probably be API

File: README.md [hunk 3]
```diff
@@ -1,11 +1,32 @@
 # Content Sources
 
 ## What is it?
+<<<<<<< HEAD
 
+=======
+>>>>>>> b899aa1 (Add code layout descriptions)
```
Comment (rverdile) [inline README.md:7]: there's some merge conflict text leftover

File: README.md [hunk 4]
```diff
@@ -1,11 +1,32 @@
...
 Content Sources is an application for storing information about external content (currently YUM repositories) in a central location.
 
 
 ## Developing
+<<<<<<< HEAD
+=======
```
Comment (rverdile) [inline README.md:13]: here too

File: README.md [hunk 5]
```diff
@@ -78,11 +99,26 @@ $ make openapi
...
+|-----------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [api](./api/)                     | Openapi docs and doc generation code                                                                                                                                                            |
+| [db/migrations](./db/migrations/) | Database Migrations                                                                                                                                                                             |                                                                                                                                                                            |
+| [pkg/api](./pkg/api)              | API Structures that are used for handling data within our Api Handlers                                                                                                                          |
```
Comment (rverdile) [inline README.md:108]: Api Handler -> API Handlers

File: README.md [hunk 6]
```diff
@@ -78,11 +99,26 @@ $ make openapi
...
+| [pkg/config](./pkg/config)        | Config loading and application bootstrapping code                                                                                                                                               |
+| [pkg/dao](./pkg/dao)              | Database Access Object.  Abstraction layer that provides an interface and implements it for our default database provider (postgresql).  It is separated out for abstraction and easier testing |
+| [pkg/db](./pkg/db)                | Database connection and migration related code                                                                                                                                                  |
+| [pkg/handler](./pkg/handler)      | Methods that directly handle Api requests                                                                                                                                                       |
```
Comment (rverdile) [inline README.md:112]: Api requests -> API requests

File: README.md [hunk 7]
```diff
@@ -1,12 +1,14 @@
...
 
 ## What is it?
-
 Content Sources is an application for storing information about external content (currently YUM repositories) in a central location.
 
 
 ## Developing
 
+### Requirements:
+1. podman installed or docker docker installed (and running)
```
Comment (rverdile) [inline README.md:10]: says "docker docker" instead of "docker"

Comment (jlsherrill) [reply README.md:10]: roger roger!

**Review by rverdile — APPROVED** [review]
great!

**Pull request merged**